### PR TITLE
docs: clarify preset file name recommendations

### PR DESCRIPTION
**Pull request by rarkins**
## Changes

Clarify preset file naming restrictions

## Context

#28434

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/config-presets.md [hunk 1]
```diff
@@ -42,6 +33,17 @@ Alternatively, Renovate can fetch preset files from an HTTP server.
 
 You can set a Git tag (like a SemVer) to use a specific release of your shared config.
 
+### Preset File Naming
+
+Presets are repo-hosted, and you can have one or more presets hosted per repository.
+If you omit a file name from your preset (e.g. `github>abc/foo`) then Renovate will look for a `default.json` file in the repo.
+If you wish to have an alternative file name, you need to specify it (e.g. `github>abc/foo:alternative-name.json5`).
```
Comment (viceice) [inline docs/usage/config-presets.md:40]:
```suggestion
If you wish to have an alternative file name, you need to specify it (e.g. `github>abc/foo//alternative-name.json5`).
```
I think we need to use this path syntax for file extension presets

Comment (viceice) [reply docs/usage/config-presets.md:40]: yes, we should remove the subpresets stuff from docs to soft deprecating them

Comment (HonkingGoose) [reply docs/usage/config-presets.md:40]: It took a lot of effort to get those tables in there and correct. Please make sure we _really_ don't need them anymore, before dropping them. 😄

I know we can always get the tables back again from the Git commit history later, if we need, but that's extra work.

File: docs/usage/config-presets.md [hunk 2]
```diff
@@ -42,6 +33,17 @@ Alternatively, Renovate can fetch preset files from an HTTP server.
...
+
+<!-- prettier-ignore -->
```
Comment (HonkingGoose) [inline docs/usage/config-presets.md:42]: This PR seems related to the table as well. So if you're going to drop the table, maybe that PR needs changes also?

- https://github.com/renovatebot/renovate/pull/28256

**Review by viceice — APPROVED** [review]

**Pull request merged**